cob_bringup/tools/docker_control.launch

```diff
@@ -32,4 +33,8 @@
 	<node pkg="cob_docker_control" type="docker_control.py" respawn="false" name="docker_control" output="screen" unless="$(arg multi_station)">
 	</node>
```

Comment:
Do you see any error message about "Node not there"?

Please remove the node from the launch file